src/com/dotcms/api/system/user/UserServiceFactory.java

```diff
@@ -0,0 +1,366 @@
+				this.includeDefault = includeDefault;
+			}
+
+			/**
+			 * Executes this filtering template based on the specified search
+			 * criteria.
+			 * 
+			 * @return A {@link Map} with the result of the filtering query.
+			 */
+			public Map<String, Object> perform() {
+				ArrayList<Map<String, String>> list = null;
+				Map<String, Object> results = new HashMap<String, Object>(2);
+				int totalItemCount = 0;
+				List<User> users = null;
+				int realUserCount = 0;
+				try {
+					// Step 1. Retrieve users, beginning from "start" parameter,
+					// up to a number of "limit" items, filtered by "filter"
+					// parameter.
+					totalItemCount = getUserCount();
+					if (start < totalItemCount) {
+						users = getUsers();
+						realUserCount = users.size();
+					}
+					// Step 2. Assemble all of this information into an
+					// appropriate container to the view
+					if (users != null) {
+						int pageSize = realUserCount;
+						list = new ArrayList<Map<String, String>>(pageSize);
+						for (User aUser : users) {
+							Map<String, String> aRecord = new HashMap<String, String>();
+							String fullName = aUser.getFullName();
+							fullName = (UtilMethods.isSet(fullName) ? fullName : " ");
+							String emailAddress = aUser.getEmailAddress();
+							emailAddress = (UtilMethods.isSet(emailAddress) ? emailAddress : " ");
+							aRecord.put("id", aUser.getUserId());
+							aRecord.put("type", USER_TYPE_VALUE);
+							aRecord.put("name", fullName);
+							aRecord.put("emailaddress", emailAddress);
+							list.add(aRecord);
+						}
+					} else {
+						list = new ArrayList<Map<String, String>>(0);
+					}
+				} catch (Exception ex) {
+					Logger.warn(UserAjax.class, "::processUsersList -> Could not process list of users.");
+					list = new ArrayList<Map<String, String>>(0);
+				}
+				results.put("data", list);
+				results.put("total", totalItemCount);
+				return results;
+			}
+		}
+
+		@Override
+		public Map<String, Object> getUsersList(String assetInode, String permission, Map<String, String> params)
+				throws Exception {
+			int start = 0;
+			if (params.containsKey("start")) {
+				start = Integer.parseInt((String) params.get("start"));
+			}
+			int limit = -1;
+			if (params.containsKey("limit")) {
+				limit = Integer.parseInt((String) params.get("limit"));
+			}
+			String query = "";
+			if (params.containsKey("query")) {
+				query = (String) params.get("query");
+			}
+			boolean includeAnonymous = false;
+			if (params.containsKey("includeAnonymous")) {
+				includeAnonymous = Boolean.valueOf((String) params.get("includeAnonymous"));
+			}
+			// Initially set to "true" for backwards compatibility
+			boolean includeDefault = true;
+			if (params.containsKey("includeDefault")) {
+				includeDefault = Boolean.valueOf((String) params.get("includeDefault"));
+			}
+			Map<String, Object> results;
+			if ((InodeUtils.isSet(assetInode) && !assetInode.equals("0"))
```

Comment:
it is safer to use the constant in the left side, such as !"0".equals (assetInode) you can even remote the isSet, since if it is null will be false :)